### ISSUE TITLE
amazon-chroot: unexpected EOF

### ISSUE THREAD

**Issue by cultavix** [mitchellh/packer#4482]
**Packer Version:** 0.12.2

**Platform:** Ubuntu 14.04

**Debug Log:** Doing this now

**Example code:** https://gist.github.com/cultavix/0e67d5ac317737db44a00a926551dbd1

```
==> Some builds didn't complete successfully and had errors:
--> amazon-chroot: unexpected EOF

==> Builds finished but no artifacts were created.
panic: interface conversion: interface {} is nil, not map[string][]string
2017/01/30 07:35:20 packer: 
2017/01/30 07:35:20 packer: goroutine 26 [running]:
2017/01/30 07:35:20 packer: panic(0x13e6f60, 0xc420370c40)
2017/01/30 07:35:20 packer: 	/usr/local/Cellar/go/1.7.3/libexec/src/runtime/panic.go:500 +0x1a1
2017/01/30 07:35:20 packer: github.com/mitchellh/packer/builder/amazon/common.(*StepAMIRegionCopy).Run(0xc420129980, 0x1e117a0, 0xc4201298f0, 0x0)
2017/01/30 07:35:20 packer: 	/Users/mwhooker/go/src/github.com/mitchellh/packer/builder/amazon/common/step_ami_region_copy.go:26 +0x8ff
2017/01/30 07:35:20 packer: github.com/mitchellh/packer/vendor/github.com/mitchellh/multistep.(*BasicRunner).Run(0xc42012ce40, 0x1e117a0, 0xc4201298f0)
2017/01/30 07:35:20 packer: 	/Users/mwhooker/go/src/github.com/mitchellh/packer/vendor/github.com/mitchellh/multistep/basic_runner.go:70 +0x207
2017/01/30 07:35:20 packer: github.com/mitchellh/packer/builder/amazon/chroot.(*Builder).Run(0xc42024a000, 0x1e140e0, 0xc42012ee40, 0x1e0d0a0, 0xc42012a850, 0x1e12e60, 0xc4201300d0, 0xc4201293b0, 0x0, 0x0, ...)
2017/01/30 07:35:20 packer: 	/Users/mwhooker/go/src/github.com/mitchellh/packer/builder/amazon/chroot/builder.go:275 +0xd7b
2017/01/30 07:35:20 packer: github.com/mitchellh/packer/packer/rpc.(*BuilderServer).Run(0xc42012ebe0, 0x1, 0xc42012a7b0, 0x0, 0x0)
2017/01/30 07:35:20 packer: 	/Users/mwhooker/go/src/github.com/mitchellh/packer/packer/rpc/builder.go:93 +0x218
2017/01/30 07:35:20 packer: reflect.Value.call(0xc42026a840, 0xc4201300c0, 0x13, 0x15ffd31, 0x4, 0xc4201c1ed0, 0x3, 0x3, 0x0, 0x0, ...)
2017/01/30 07:35:20 packer: 	/usr/local/Cellar/go/1.7.3/libexec/src/reflect/value.go:434 +0x5c8
2017/01/30 07:35:20 packer: reflect.Value.Call(0xc42026a840, 0xc4201300c0, 0x13, 0xc4201c1ed0, 0x3, 0x3, 0xc4202d4c58, 0x1, 0x0)
2017/01/30 07:35:20 packer: 	/usr/local/Cellar/go/1.7.3/libexec/src/reflect/value.go:302 +0xa4
2017/01/30 07:35:20 packer: net/rpc.(*service).call(0xc42012c8c0, 0xc42012c880, 0xc42012a770, 0xc420146800, 0xc4201b5860, 0x13117c0, 0xc42012a77c, 0x18a, 0x12e6d00, 0xc42012a7b0, ...)
2017/01/30 07:35:20 packer: 	/usr/local/Cellar/go/1.7.3/libexec/src/net/rpc/server.go:383 +0x148
2017/01/30 07:35:20 packer: created by net/rpc.(*Server).ServeCodec
2017/01/30 07:35:20 packer: 	/usr/local/Cellar/go/1.7.3/libexec/src/net/rpc/server.go:477 +0x421
2017/01/30 07:35:20 ui error: Build 'amazon-chroot' errored: unexpected EOF
2017/01/30 07:35:20 Builds completed. Waiting on interrupt barrier...
2017/01/30 07:35:20 /usr/local/bin/packer: plugin process exited
2017/01/30 07:35:20 machine readable: error-count []string{"1"}
2017/01/30 07:35:20 ui error: 
==> Some builds didn't complete successfully and had errors:
2017/01/30 07:35:20 machine readable: amazon-chroot,error []string{"unexpected EOF"}
2017/01/30 07:35:20 ui error: --> amazon-chroot: unexpected EOF
2017/01/30 07:35:20 ui: 
==> Builds finished but no artifacts were created.
2017/01/30 07:35:20 waiting for all plugin processes to complete...
2017/01/30 07:35:20 /usr/local/bin/packer: plugin process exited
2017/01/30 07:35:20 /usr/local/bin/packer: plugin process exited
2017/01/30 07:35:20 /usr/local/bin/packer: plugin process exited
2017/01/30 07:35:20 /usr/local/bin/packer: plugin process exited
2017/01/30 07:35:20 /usr/local/bin/packer: plugin process exited



!!!!!!!!!!!!!!!!!!!!!!!!!!! PACKER CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!

Packer crashed! This is always indicative of a bug within Packer.
A crash log has been placed at "crash.log" relative to your current
working directory. It would be immensely helpful if you could please
report the crash with Packer[1] so that we can fix this.

[1]: https://github.com/mitchellh/packer/issues

!!!!!!!!!!!!!!!!!!!!!!!!!!! PACKER CRASH !!!!!!!!!!!!!!!!!!!!!!!!!!!!
Command exited with non-zero status 1
54.23user 29.19system 3:14.03elapsed 42%CPU (0avgtext+0avgdata 204236maxresident)k
123762inputs+423592outputs (305major+11786296minor)pagefaults 0swaps
Packer completed with status 1. Exiting now.
AMI has not been created. Exit now.
```

**Comment by cultavix** [comment]
**Configuration:** https://gist.github.com/cultavix/0e67d5ac317737db44a00a926551dbd1

**Comment by rickard-von-essen** [comment]
This looks like a duplicate of #4340 which was fixed in 0.12.2, cloud you provide a full log and verify the version you have installed.

**Comment by cultavix** [comment]
it's definitely version `0.12.2.`

I'll re-run it now again to see if I get more output

**Comment by mwhooker** [comment]
Sounds like it's fixed. Please open a new issue if you can confirm that this is still broken in versions > 12.1
